cliconfig/config.go

```diff
@@ -15,19 +15,19 @@ import (
 )
 
 const (
-	// Where we store the config file
-	CONFIGFILE     = "config.json"
-	OLD_CONFIGFILE = ".dockercfg"
+	// CONFIGFILE is the name of config file
+	CONFIGFILE    = "config.json"
```

Comment:
This is the only exported usage I see.
registry/auth_test.go
37:	root = filepath.Join(root, cliconfig.CONFIGFILE)